rules/windows/discovery_privileged_localgroup_membership.toml

```
@@ -14,13 +14,67 @@ index = ["winlogbeat-*", "logs-system.*"]
+After a successful initial compromise, attackers may try to gain situational awareness of the environment to plan their
+next steps. This can happen by running commands to enumerate network resources, users, connections, files, and security
+software installed.
+
+This rule looks for the enumeration of privileged local groups membership by suspicious processes, excluding known legit
+utilities and programs installed. This information can be used by attackers to decide the next steps of the attack, such
+as mapping targets for credential compromise and other post-exploitation activities.
+
+#### Possible investigation steps
+
+- Identify the process, host and user involved on the event.
+- Investigate the process execution chain (parent process tree).
+- Investigate other alerts associated with the user/host during the past 48 hours.
+- Investigate abnormal behaviors done using the account, such as commands executed, startup folder modifications, and
+network connections.
+- Retrieve the process executable and determine if it is malicious:
+  - Check if the file belongs to the operating system or has a valid digital signature.
+  - Use a private sandboxed malware analysis system to perform analysis.
+    - Observe and collect information about the following activities:
+      - Attempts to contact external domains and addresses.
+      - File and registry access, modification, and creation activities.
+      - Service creation and launch activities.
+      - Scheduled tasks creation.
+
+### False positive analysis
+
+- Discovery activities themselves are not inherently malicious if occurring in isolation. As long as the analyst did not
+identify any suspicious activity related to the user or host, such alerts can be dismissed.
+- If this activity is expected and noisy in your environment, consider adding exceptions — preferably with a combination
+of user and command line conditions.
+
+### Response and remediation
 
+- Initiate the incident response process based on the outcome of the triage.
+- Quarantine the involved host to prevent further post-compromise behavior.
+- Reset passwords for the user account and other potentially compromised accounts (email, services, CRMs, etc.).
+- Determine the initial infection vector.
+
+## Config
+
+The 'Audit Security Group Management' audit policy must be configured (Success).
+Steps to implement the logging policy with with Advanced Audit Configuration:
+
+```
+Computer Configuration > 
+Policies > 
+Windows Settings > 
+Security Settings > 
+Advanced Audit Policies Configuration > 
+Audit Policies > 
+Account Management >
+Audit Security Group Management (Success)
+```
+
+Microsoft introduced the event used in this detection rule on Windows 10/2016 and later operating systems.
```

Edited to use Microsoft's [versioning syntax](https://docs.microsoft.com/en-us/windows/release-health/release-information):
```suggestion
Microsoft introduced the event used in this detection rule on Windows 10, version 1607 and later operating systems.
```

---

@w0rk3r Good call, my previous edit left out Windows Server; "1607" only refers to a version of Windows 10. How about mentioning them both then?

```
Microsoft introduced the event used in this detection rule in Windows 10 version 1607, Windows Server 2016, and later operating systems.
```